lib/util/request.ts

```diff
@@ -15,11 +18,19 @@ const proxyError = (result): never => {
   throw result;
 };
 
+const userAgent = `TaxJar/Node (${[
```

Comment:
This might work better as a `get` instead with this broken into a few lines. This one liner is a bit tough to digest for me.

```
    get userAgent(): string {
        const osVersion = '';
        // more ...
    }
```

Reply:
Good feedback! 👍 A new `getUserAgent` function includes variable names so hopefully it will be easier to parse. For simplicity, I've refactored to a function rather than a private method.